node/packages/parser/src/ethereum/yearn.ts

```diff
@@ -0,0 +1,33 @@
+import { Tx } from '@shapeshiftoss/blockbook'
+import { TxSpecific as ParseTxSpecific, YearnTx } from '../types'
+import shapeShiftRouter from './abi/shapeShiftRouter'
+import yearnVault from './abi/yearnVault'
+import { addABI, decodeMethod } from 'abi-decoder'
+import { GenericParser } from './index'
+
+const ROUTER_CONTRACT = '0x6a1e73f12018D8e5f966ce794aa2921941feB17E'
+
+export class Parser implements GenericParser {
+  supportsTransaction(tx: Tx) {
+    const data = tx.ethereumSpecific?.data
+    const firstVoutAddress = tx.vout && tx.vout[0].addresses && tx.vout[0].addresses[0]
```

Comment:
Prefer optional chaining over &&ing, and as far as blockbook is concerned, this is always the receive address to better describe value vs `firstVoutAddress` if wanting to set to variable.

Reply:
Nice, it slipped my mind that we could use optional chaining on array accessors. Addressed in https://github.com/shapeshift/unchained/pull/267/commits/2953d65c9a978276a520a2f9707dcb3c1381583e.